tests/queries/0_stateless/02118_deserialize_whole_text.sh

```diff
@@ -33,10 +33,10 @@ echo "[\"255.255.255.255trash\"]" > $DATA_FILE
 $CLICKHOUSE_CLIENT -q "SELECT * FROM file('data_02118', 'JSONCompactStringsEachRow', 'x IPv4')" 2>&1 | grep -F -q "UNEXPECTED_DATA_AFTER_PARSED_VALUE" && echo 'OK' || echo 'FAIL'
 
 echo "255.255.255.255trash" > $DATA_FILE
-$CLICKHOUSE_CLIENT -q "SELECT * FROM file('data_02118', 'TSV', 'x IPv4')" 2>&1 | grep -F -q "UNEXPECTED_DATA_AFTER_PARSED_VALUE" && echo 'OK' || echo 'FAIL'
+$CLICKHOUSE_CLIENT -q "SELECT * FROM file('data_02118', 'TSV', 'x IPv4')" 2>&1 | grep -F -q "CANNOT_PARSE_INPUT_ASSERTION_FAILED" && echo 'OK' || echo 'FAIL'
```

Comment:
Why error code is not equal to `UNEXPECTED_DATA_AFTER_PARSED_VALUE` ?

Reply:
I see that previously exception was from `SerializationIP` (which tried to parse a full string, and probably it lead to a bug from parser).
Now, exceptions are from `TabSeparatedFormatReader::skipRowEndDelimiter` (or other formats). Looks ok.

Reply:
Agreed.